bip-0301.mediawiki

```diff
@@ -12,181 +12,149 @@
   License: BSD-2-Clause
 </pre>
 
-==Abstract==
```

Comment:
Please keep to BIP 2-specified format. Abstract should be present.

Reply:
OK, will do